Pilot/pilotTools.py

```diff
@@ -415,8 +435,8 @@ def getCommand(params, commandName):
     """Get an instantiated command object for execution.
     Commands are looked in the following modules in the order:
 
-    1. <CommandExtension>Commands
-    2. pilotCommands
+    1. `<CommandExtension>Commands`
+    2. `pilotCommands`
```

Comment:
I think actually it is:
```suggestion
    1. `<CommandExtension>PilotCommands`
    2. `PilotCommands`
```